nixos/modules/services/networking/gogoclient.nix

```diff
@@ -34,7 +34,7 @@ in
 
       password = mkOption {
         default = "";
-        type = types.string;
+        type = types.str;
```

Comment:
see https://github.com/NixOS/nixpkgs/pull/66346/files#r312350479

Reply:
Left this as `types.str` because the default is `""`